inception-project/inception
# #836 - Full text search IRI not set from Knowledge Base Profile

**What's in the PR**
* Fix for #836 

**How to test manually**
* Create new knowledge base and select a profile with pre-defined full-text-search IRI
* See if full-text-search IRI is set properly

**Automatic testing**

**Documentation**
